hipchat.go

```diff
@@ -168,10 +183,20 @@ func (c *Client) RoomHistory(id, date, tz string) ([]Message, error) {
 	uri := fmt.Sprintf("%s/rooms/history?auth_token=%s&room_id=%s&date=%s&timezone=%s",
 		c.BaseURL, url.QueryEscape(c.AuthToken), url.QueryEscape(id), url.QueryEscape(date), url.QueryEscape(tz))
 
-	resp, err := http.Get(uri)
+	reqs, err := http.NewRequest("GET", uri, nil)
```

Comment:
i changed ``req`` ==> ``reqs`` because ``req`` is already used by MessageRequest. We need to change either of the object name.
or may be i can use ``hreq`` for http type of request object.